python/pyarrow/tests/test_parquet.py

```diff
@@ -91,8 +87,14 @@ def test_pandas_parquet_2_0_rountrip(tmpdir):
 
     filename = tmpdir.join('pandas_rountrip.parquet')
     arrow_table = pa.Table.from_pandas(df, timestamps_to_ms=True)
+    expected_custom_metadata = {
+        b'indices': bytes([len(df.columns)]),
```

Comment:
This limits us to `MultiIndex`es with <= 255 levels. I think that's reasonable for now. We can always come with up a more complex encoding if we want to support more levels than that. I'd be surprised if this ever comes up in practice.